compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/transform/CreateViewTransform.java

```diff
@@ -52,8 +56,11 @@ public void prepare(final Context context, final OutputCollector<O> oc) {
 
   @Override
   public void onData(final I element) {
-    WindowedValue<I> data = WindowedValue.valueInGlobalWindow(element);
-    windowed.add(data);
+    // Since CreateViewTransform takes KV(Void, value), this is okay
+    if (element instanceof KV) {
+      final KV<?, ?> kv = (KV<?, ?>) element;
+      multiView.getDataList().add(kv.getValue());
+    }
```

Comment:
Just to double-check, can you do something like

else {
  throw new IllegalStateException();
}

Reply:
For handling sideInputs, I will look carefully again and see if I can make it more concise. Thanks for the comments.